### PR TITLE
Adding multiweight support for inception prototype model 

### LINKED ISSUE #4672: Multi-pretrained weights: Add model builders for Classification
### 🚀 The feature

Update the model builders for all Classification models:
- [x] AlexNet
- [x] DenseNet
- [x] EfficientNet
- [x] GoogleNet
- [x] InceptionV3
- [x] MNASNet
- [x] MobileNetV2
- [x] MobileNetV3
- [x] RegNet
- [x] ResNet
- [x] ShuffleNetV2
- [x] SqueezeNet
- [x] VGG

cc @datumbox @vfdev-5 @pmeier @bjuncek

### PR DESCRIPTION
**Pull request by datumbox**
Fixes #4672

cc @datumbox @bjuncek

### CODE REVIEW

File: torchvision/models/inception.py [hunk 1]
```diff
@@ -26,43 +26,6 @@
 _InceptionOutputs = InceptionOutputs
 
 
-def inception_v3(pretrained: bool = False, progress: bool = True, **kwargs: Any) -> "Inception3":
```
Comment (datumbox) [inline torchvision/models/inception.py:29]: Moving the builder at the bottom of the file to use proper typing.

File: torchvision/prototype/models/inception.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    )
+
+
+def inception_v3(weights: Optional[Inception3Weights] = None, progress: bool = True, **kwargs: Any) -> Inception3:
+    if "pretrained" in kwargs:
+        warnings.warn("The argument pretrained is deprecated, please use weights instead.")
+        weights = Inception3Weights.ImageNet1K_TFV1 if kwargs.pop("pretrained") else None
+    weights = Inception3Weights.verify(weights)
+
+    original_aux_logits = kwargs.get("aux_logits", True)
```
Comment (datumbox) [inline torchvision/prototype/models/inception.py:38]: Similar to GoogleNet but here the default value is True not False

**Review by datumbox — COMMENTED** [review]
Few clarifications:

**Review by jdsgomes — APPROVED** [review]
LGTM. Thanks!
